charts/hedera-mirror/templates/secret-timescaledb-credential.yaml

```
@@ -0,0 +1,15 @@
+{{- if .Values.timescaledb.enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    {{- include "hedera-mirror.labels" . | nindent 4 }}
+  name: {{ printf "%s-credentials" .Release.Name }}
+  namespace: {{ include "hedera-mirror.namespace" . }}
+type: Opaque
+stringData:
+  # base64 encoded password (default is mirror_node_pass)
+  PATRONI_SUPERUSER_PASSWORD: "bWlycm9yX25vZGVf1cGFzcwo="
+  PATRONI_REPLICATION_PASSWORD: "bWlycm9yX25vZGVfcGFzcwo="
+  PATRONI_admin_PASSWORD: "bWlycm9yX25vZGVfcGFzcwo="
```

These don't [seem](https://github.com/timescale/timescaledb-kubernetes/blob/88d1bb40521c93feed3b17785d7d1d5c1f56669e/charts/timescaledb-single/generate_kustomization.sh#L46) to be base64 as you commented. The entire secret itself will be base64 when applied since you're using `stringData`.

```suggestion
  {{- $password := .Values.importer.config.hedera.mirror.importer.db.password | quote }}
  PATRONI_SUPERUSER_PASSWORD: {{ $password }}
  PATRONI_REPLICATION_PASSWORD: {{ $password }}
  PATRONI_admin_PASSWORD: {{ $password }}
```

---

Yeah, I misunderstood their admin guide to mean what was going into the secret instead of what was coming out.  Added this change.